YellowNotes.Core/Services/EmailService.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System.Threading.Tasks;
+using System.Threading;
+using YellowNotes.Core.Email;
+using MimeKit;
+using MailKit.Net.Smtp;
+
+namespace YellowNotes.Core.Services
+{
+    public class EmailService : IEmailService
+    {
+        private readonly EmailConfiguration _emailConfiguration;
```

Comment:
Let's stick to a consistent naming convention that is already present in this repository. I don't like this one for various reasons, many of which are described [here](https://stackoverflow.com/a/17937309).